pkg/template/template.go

```diff
@@ -0,0 +1,68 @@
+package template
+
+import (
+	"errors"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"regexp"
+	"strings"
+)
+
+var (
+	ErrCouldNotReadFile = errors.New("could not read file")
```

Comment:
Is this worth duplicating [here](https://github.com/operator-framework/combo/pull/38/files#diff-56eae6d32e732794aa20bcb63594976b8c9780b628767d3da1e8f85843120fbfR20)?

Reply:
I made a new `comboErrors` package for errors that are shared across Combo.